src/processes/registerv2-process.js

```diff
@@ -70,10 +70,11 @@ const registerProcess = async (job, done) => {
     await ProcessHelper.followAnonymousUser(anonUserId, follows);
     await ProcessHelper.followTopic(userId, topics);
     await ProcessHelper.followLocation(userId, result?.locations);
-    await ProcessHelper.addUserToTopicChannel(userId, topicNames);
+    await ProcessHelper.sendTopicAutoMessage(userId, topicIds);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the order of function calls might have implications. If `sendTopicAutoMessage` relies on the user being added to the topic channel first (which was the case in the old code), this could potentially cause issues.
```diff
-    await ProcessHelper.sendTopicAutoMessage(userId, topicIds);
+    await ProcessHelper.addUserToTopicChannel(userId, topicNames);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->